clang/include/clang/Sema/Sema.h

```diff
@@ -10639,9 +10639,9 @@ class Sema final : public SemaBase {
   /// BuildOverloadedArrowExpr - Build a call to an overloaded @c operator->
   ///  (if one exists), where @c Base is an expression of class type and
   /// @c Member is the name of the member we're trying to find.
-  ExprResult BuildOverloadedArrowExpr(Scope *S, Expr *Base,
-                                      SourceLocation OpLoc,
-                                      bool *NoArrowOperatorFound = nullptr);
+  ExprResult BuildOverloadedArrowExpr(Expr *Base, SourceLocation OpLoc,
+                                      bool *NoArrowOperatorFound,
+                                      bool &IsDependent);
```

Comment:
Ooof, I'm almost always really against 'out' parameters in C++.  I'd hope we could come up with a better way for this.